### ISSUE TITLE
Redesign "Volume Group" UI

### ISSUE THREAD

**Issue by alinefm** [kimchi-project/ginger#301]
There are some issues in the Volume Group section:

1. Why multiple selection is enabled if no action can be taken?

2. Paging should be shown only when needed

3. Align Refresh button and Filter input box.

![4f48e5bfb02135ce](https://cloud.githubusercontent.com/assets/4921718/14303972/5e06342c-fb85-11e5-821b-1c4b9e43c905.jpg)

Said that, my suggestion is to change the Volume Group section to something like below:

```
- Volume Group

| Refresh |              | Filter                   |

| Name                 | Capacity                   |
| vg1                  | 10GB                       |
| vg2                  | 12GB                       |
```

All widgets shown on mockup MUST follow Wok style.

**Comment by samhenri** [comment]
@alinefm should we keep jQuery Bootgrid or use default wok.datagrid? Do we have confirmation that multi-selection is not necessary?

**Comment by alinefm** [comment]
Fixed by 82d17adfc840b783ebb2e37e53a7e9109a7bfad1
